lib/rails_admin_import/importer.rb

```diff
@@ -0,0 +1,224 @@
+require "rails_admin_import/import_logger"
+
+module RailsAdminImport
+  class Importer
+    def initialize(import_model, params)
+      @import_model = import_model
+      @params = params
+    end
+
+    attr_reader :import_model, :params
+
+    class UpdateLookupError < StandardError
+    end
+
+    def import(records)
+      begin
+        init_results
+
+        # TODO: re-implement file size check
+        # if file_check.readlines.size > RailsAdminImport.config.line_item_limit
+        #   return results = { :success => [], :error => ["Please limit upload file to #{RailsAdminImport.config.line_item_limit} line items."] }
+        # end
+
+        with_transaction do
+          records.each do |record|
+            import_record(record)
+          end
+
+          rollback_if_error
+        end
+      rescue Exception => e
```

Comment:
Avoid rescuing the `Exception` class. Perhaps you meant to rescue `StandardError`?